src/main/__tests__/refreshTokens.spec.ts

```diff
@@ -36,6 +37,16 @@ test('simple', async () => {
     })
   )
 
+  const refreshCallWithRefreshToken = fetchMock.mockResponseOnce(
```

Comment:
Prefer defining a new test case and rename tests where necessary.

Reply:
split into two tests